src/utils/socket/index.js

```diff
@@ -0,0 +1,11 @@
+import io from '@/package/socket';
+
+const socketUtils = {
+  _instance: undefined,
+  getConnection() {
+    if (!this._instance) this._instance = io('http://localhost:3001/');
+    return this._instance;
+  },
+};
```

Comment:
하.. 또 코드리뷰가 날라갔어요.. 다시 작성합니다 ㅠㅠ 

Nit: 아래와 같은 코드는 어떨까요? `if(this._instance)`와 확인 로직이 필요해지지 않을 것 같아서요. 아래 코드를 작성하면서 자바스크립트에서 싱글톤을 구현하는 방법을 찾아봤는데 `es7`의 `static`을 사용해서 단 하나의 인스턴스를 보장하는 방법도 있더라구요. 지금 코드도 좋지만 다른 코드도 보시면 좋을 거 같아서 링크 남깁니다!

```suggestion

const LOCAL_SERVER = 'http://localhost:3001/';
const DEV_SERVER = 'http://jinyoung-odroid.iptime.org:3001/';

class Socket {
  constructor() {
    this._instance = io(LOCAL_SERVER);
    return this._instance;
  }
};

export const SocketService = new Socket();
```

Reply:
혹시 위 코드에서 `this._instance`를 사용하면 저절로 `static`으로 선언되는 건가요?

Reply:
앗 아뇨! 위 코드는 제가 작성한 코드인데 `static` 사용하지 않고 `export default new Socket()`로 싱글톤을 구현한 것이고 `static`를 사용한 방법도 있길래 링크 걸어놨습니다!